nav2_route/src/plugins/graph_file_savers/geojson_graph_file_saver.cpp

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) 2024 Leidos
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+#include <string>
+#include <vector>
+#include <fstream>
+
+#include "nav2_route/plugins/graph_file_savers/geojson_graph_file_saver.hpp"
+
+namespace nav2_route
+{
+
+void GeoJsonGraphFileSaver::configure(
+  const rclcpp_lifecycle::LifecycleNode::SharedPtr node)
+{
+  RCLCPP_INFO(node->get_logger(), "Configuring geojson graph file saver");
+  logger_ = node->get_logger();
+}
+
+bool GeoJsonGraphFileSaver::saveGraphToFile(
+  Graph & graph, std::string filepath)
+{
+  Json json_graph, json_crs, json_properties;
```

Comment:
Can we put this in a try catch? streaming data from a file is likely to run into issues.